### DISCUSSION TITLE
Brand new website from Basic starter template fails Netlify build

### DISCUSSION THREAD

**Discussion by samgermain** [gatsbyjs/gatsby#36745] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

The basic gatsby starter fails to build on Netlify right from the beginning. I add nothing to the site, and build the site from Github, and receive the error response below. 

### Reproduction Link

https://github.com/samgermain/test_repo

### Steps to Reproduce

1. run `gatsby new` and use these settings
```
What would you like to call your site?
✔ · test-repo
What would you like to name the folder where your site will be created?
✔ dev/ test_repo
✔ Will you be using JavaScript or TypeScript?
· JavaScript
✔ Will you be using a CMS?
· Netlify CMS
✔ Would you like to install a styling system?
· Sass
✔ Would you like to install additional features with other plugins?
· Add the Google Analytics tracking script
· Add responsive images
· Add page meta tags with React Helmet
· Add an automatic sitemap
· Generate a manifest file
```
2. cd test_repo
3. git remote add origin https://github.com/your_username/test_repo
4. On netlify.com 
    - login
    - click "add a new site" -> "import an existing project"
        - ![](https://i.imgur.com/DHZrNWy.png)
    - select a project
        - ![](https://i.imgur.com/fJhVHf2.png)
    - change the **build command** to `gatsby build` and click **deploy site**
        - ![](https://i.imgur.com/JO04Kpg.png)
...

### Expected Result

The site should build on netlify

### Actual Result

9:47:10 PM: Build ready to start
9:47:12 PM: build-image version: 9289b698a503f132b265bd089ab40ce1d6f9e65e (focal)
9:47:12 PM: build-image tag: v4.12.0
9:47:12 PM: buildbot version: 8004afba0d79c2913deefb1ea0ee8f3ed710b828
9:47:12 PM: Fetching cached dependencies
9:47:12 PM: Failed to fetch cache, continuing with build
9:47:12 PM: Starting to prepare the repo for build
9:47:12 PM: No cached dependencies found. Cloning fresh repo
9:47:12 PM: git clone https://github.com/samgermain/test_repo
9:47:13 PM: Preparing Git Reference refs/heads/master
9:47:13 PM: Parsing package.json dependencies
9:47:14 PM: Starting build script
9:47:14 PM: Installing dependencies
9:47:14 PM: Python version set to 2.7
9:47:15 PM: v16.17.1 is already installed.
9:47:15 PM: Now using node v16.17.1 (npm v8.15.0)
9:47:15 PM: Started restoring cached build plugins
9:47:15 PM: Finished restoring cached build plugins
9:47:15 PM: Attempting ruby version 2.7.2, read from environment
9:47:16 PM: Using ruby version 2.7.2
9:47:16 PM: Using PHP version 8.0
9:47:16 PM: No npm workspaces detected
9:47:16 PM: Started restoring cached node modules
9:47:16 PM: Finished restoring cached node modules
9:47:17 PM: Installing NPM modules using NPM version 8.15.0
9:47:17 PM: npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
9:47:17 PM: npm WARN config location in the cache, and they are managed by
9:47:17 PM: npm WARN config     [`cacache`](http://npm.im/cacache).
9:47:17 PM: npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
9:47:17 PM: npm WARN config location in the cache, and they are managed by
9:47:17 PM: npm WARN config     [`cacache`](http://npm.im/cacache).
9:47:18 PM: npm ERR! code ERESOLVE
9:47:18 PM: npm ERR! ERESOLVE could not resolve
9:47:18 PM: npm ERR!
9:47:18 PM: npm ERR! While resolving: @mdx-js/react@1.6.22
9:47:18 PM: npm ERR! Found: react@18.1.0
9:47:18 PM: npm ERR! node_modules/react
9:47:18 PM: npm ERR!   react@"^18.1.0" from the root project
9:47:18 PM: npm ERR!   peer react@">=16.3.0" from @emotion/core@10.3.1
9:47:18 PM: npm ERR!   node_modules/@emotion/core
9:47:18 PM: npm ERR!     peer @emotion/core@"^10.0.27" from @emotion/styled@10.3.0
9:47:18 PM: npm ERR!     node_modules/@emotion/styled
9:47:18 PM: npm ERR!       @emotion/styled@"^10.0.27" from netlify-cms-app@2.15.72
9:47:18 PM: npm ERR!       node_modules/netlify-cms-app
9:47:18 PM: npm ERR!         netlify-cms-app@"^2.15.72" from the root project
9:47:18 PM: npm ERR!         1 more (gatsby-plugin-netlify-cms)
9:47:18 PM: npm ERR!       17 more (netlify-cms-backend-azure, ...)
9:47:18 PM: npm ERR!     peer @emotion/core@"^10.0.28" from @emotion/styled-base@10.3.0
9:47:18 PM: npm ERR!     node_modules/@emotion/styled-base
9:47:18 PM: npm ERR!       @emotion/styled-base@"^10.3.0" from @emotion/styled@10.3.0
9:47:18 PM: npm ERR!       node_modules/@emotion/styled
9:47:18 PM: npm ERR!         @emotion/styled@"^10.0.27" from netlify-cms-app@2.15.72
9:47:18 PM: npm ERR!         node_modules/netlify-cms-app
9:47:18 PM: npm ERR!         17 more (netlify-cms-backend-azure, ...)
9:47:18 PM: npm ERR!     22 more (netlify-cms-app, netlify-cms-backend-azure, ...)
9:47:18 PM: npm ERR!   39 more (@emotion/react, @emotion/styled, @emotion/styled-base, ...)
9:47:18 PM: npm ERR!
9:47:18 PM: npm ERR! Could not resolve dependency:
9:47:18 PM: npm ERR! peer react@"^16.13.1 || ^17.0.0" from @mdx-js/react@1.6.22
9:47:18 PM: npm ERR! node_modules/@mdx-js/react
9:47:18 PM: npm ERR!   @mdx-js/react@"^1.6.22" from the root project
9:47:18 PM: npm ERR!
9:47:18 PM: npm ERR! Conflicting peer dependency: react@17.0.2
9:47:18 PM: npm ERR! node_modules/react
9:47:18 PM: npm ERR!   peer react@"^16.13.1 || ^17.0.0" from @mdx-js/react@1.6.22
9:47:18 PM: npm ERR!   node_modules/@mdx-js/react
9:47:18 PM: npm ERR!     @mdx-js/react@"^1.6.22" from the root project
9:47:18 PM: npm ERR!
9:47:18 PM: npm ERR! Fix the upstream dependency conflict, or retry
9:47:18 PM: npm ERR! this command with --force, or --legacy-peer-deps
9:47:18 PM: npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
9:47:18 PM: npm ERR!
9:47:18 PM: npm ERR! See /opt/buildhome/.npm/eresolve-report.txt for a full report.
9:47:18 PM: npm ERR! A complete log of this run can be found in:
9:47:18 PM: npm ERR!     /opt/buildhome/.npm/_logs/2022-10-05T03_47_17_555Z-debug-0.log
9:47:18 PM: Error during NPM install
9:47:18 PM: Build was terminated: Build script returned non-zero exit code: 1
9:47:18 PM: Creating deploy upload records
9:47:19 PM: Failing build: Failed to build site
9:47:19 PM: Failed during stage 'building site': Build script returned non-zero exit code: 1 (https://ntl.fyi/exit-code-1)
9:47:19 PM: Finished processing build request in 6.683949978s

### Environment

```shell
Build image
    Ubuntu Focal 20.04 (default)
```

**Comment by LekoArts** [comment]
I suggest reaching out to Netlify's support about this, they should run `npm install` with `--legacy-peer-deps`
